src/components/header/NavDropDownButton/NavDropDownButton.test.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react'
+import { render, fireEvent } from '@testing-library/react'
+
+import { NavDropDownButton } from './NavDropDownButton'
+
+const testLabel = 'Nav Label'
+
+describe('NavDropDownButton component', () => {
+  it('renders without errors', () => {
+    const onToggle = (): void => {
+      /* mock toggle fn */
+    }
+    const { getByTestId } = render(
+      <NavDropDownButton
+        label={testLabel}
+        id="testOne"
+        isOpen={true}
+        onToggle={onToggle}
+      />
+    )
+    expect(getByTestId('navDropDownButton')).toBeInTheDocument()
+  })
+
+  it('implements an onClick handler', () => {
+    const onClickFn = jest.fn()
+    const { getByText } = render(
+      <NavDropDownButton
+        label={testLabel}
+        id="testOne"
+        isOpen={true}
+        onToggle={onClickFn}
+      />
+    )
+
+    fireEvent.click(getByText(testLabel))
+    expect(onClickFn).toHaveBeenCalledTimes(1)
+  })
+
+  it('renders when isOpen is false', () => {
```

Comment:
This test text doesn't quite make sense to me- maybe its missing a word? Looks like you are testing `aria-expanded`.